projects/js-packages/components/tools/pricing-utils/test/index.ts

```diff
@@ -0,0 +1,39 @@
+import { isFirstMonthTrial } from '..';
+import { IntroOffer } from './../types';
+
+const trialIntroOffer: IntroOffer = {
+	product_id: 0,
+	product_slug: 'test_product',
+	currency_code: 'USD',
+	formatted_price: '$10',
+	original_price: 10,
+	raw_price: 1,
+	discount_percentage: 90,
+	ineligible_reason: null,
+	interval_unit: 'month',
+	interval_count: 1,
+};
```

Comment:
Not 100% sure it is necessary for i1, but having this kind of data in a `fixtures` or `data` folder would be nice. That way when new functions are added to this package, similar data could be put there